### PR TITLE
JingleSessionPC cleanup

### PR DESCRIPTION
**Pull request by paweldomas**

### CODE REVIEW

File: modules/xmpp/JingleSession.js [hunk 1]
```diff
@@ -23,21 +23,35 @@ export default class JingleSession {
...
         this.iceConfig = iceConfig;
 
+        /**
+         * Indicates whether this instance is an initiator or an answerer of
+         * the Jingle session.
+         * @type {boolean}
+         */
+        this.isInitiator = isInitiator;
+
+        this.initiator = this.isInitiator ? this.localJid : this.remoteJid;
```
Comment (bbaldino) [inline modules/xmpp/JingleSession.js]: what about naming these `initiatorJid`/`respondJid`?

File: modules/xmpp/JingleSession.js [hunk 2]
```diff
@@ -23,21 +23,35 @@ export default class JingleSession {
...
+
+        this.responder = this.isInitiator ? this.remoteJid : this.localJid;
```
Comment (saghul) [inline modules/xmpp/JingleSession.js]: why not make these getter properties?

File: modules/xmpp/JingleSessionPC.js [hunk 3]
```diff
@@ -242,48 +243,54 @@ export default class JingleSessionPC extends JingleSession {
...
+                = options.disableSimulcast
+                    || (options.preferH264 && !options.disableH264);
+            pcOptions.preferH264 = options.preferH264;
             pcOptions.enableFirefoxSimulcast
-                = this.room.options.testing
-                    && this.room.options.testing.enableFirefoxSimulcast;
-            pcOptions.enableLayerSuspension
-                = this.room.options.enableLayerSuspension;
+                = options.testing && options.testing.enableFirefoxSimulcast;
+            pcOptions.enableLayerSuspension = options.enableLayerSuspension;
```
Comment (saghul) [inline modules/xmpp/JingleSessionPC.js]: Not sure if it has any impact, but this is not going to work if there is no simulcast, is it? Maybe check for simulcast too then?

Comment (paweldomas) [reply modules/xmpp/JingleSessionPC.js]: This commit refactors the options, but I don't want to do any changes to the logic behind it.

**Comment by paweldomas** [comment]
jenkins test this please

**Pull request merged**